pkg/util/paramtable/quota_param.go

```diff
@@ -1584,15 +1584,10 @@ specific conditions, such as memory of nodes to water marker), ` + "true" + ` me
 		Version:      "2.2.0",
 		DefaultValue: defaultMaxTtDelay,
 		Formatter: func(v string) string {
-			if !p.TtProtectionEnabled.GetAsBool() {
```

Comment:
Why was this check removed?

Reply:
It is so wired to return math.MaxInt64 if TtProtectionEnabled is false, which should return a configured value.